src/run_tribler.py

```diff
@@ -135,7 +139,10 @@ def init_boot_logger():
             logger.info("Shutting down Tribler")
             if trace_logger:
                 trace_logger.close()
+
             # Flush all the logs to make sure it is written to file before it exits
             for handler in logging.getLogger().handlers:
                 handler.flush()
+
+            SentryReporter.global_strategy = SentryStrategy.SEND_SUPPRESSED
```

Comment:
Without this Sentry suppression, it sometimes happens that an error takes place during the Tribler shutdown. Then Sentry tries to display the dialog window, but the Qt state is already partially destructed, which leads to mysterious crashes.